doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,415 @@
+- Ambiguity as to whether traits with no properties should be imbued in
+  the response.
+
+## Outcome
+
+The `managementPolicy` mechanism should always be queried with the full
+trait set for any given entity specification, and any additional traits
+a host may attempt to resolve/publish for that type should be added as a
+specialisation (if known).
+
+The response populated by a manager should include all
+resolvable/persisted traits from that set, along with any additional
+traits that describe the manager's behaviour. Traits with no properties
+do not need to be imbued (see below).
+
+## Rationale
+
+Option 3 (chosen) provides the highest quality information to both hosts
+and managers, whilst keeping the additional programming overhead to a
+minimum. It allows both parties to adapt their business logic in an
+explicit and coordinated fashion.
+
+Option 1 conveys which trait sets may be managed, but not that some
+traits are unresolvable.
+
+Option 2 precludes the ability to differentiate between unmanaged trait
+sets and unresolvable component traits and so is not a viable option.
+
+## Further notes
+
+The chosen option effectively formalises the meaning of related API
+methods as follows:
+
+### `managementPolicy`
+
+For each Trait Set supplied by the Host (defining a concrete Entity type
+specialisation), the Manager will return a `TraitsData` instance imbued
+with:
+
+- Domain-specific policy traits that describe the Manager's handling of
+  entities with that trait set.
+- Any traits from the query set that have properties whose data can be
+  resolved by (read context) or registered with (write context) the
+  Manager when the supplied Entity Reference points to an entity of that
+  trait set.
+
+> Note:
+> Traits in the queried trait set with no properties will never be imbued
+> in the result. This significantly simplifies implementation in the
+> manager as it only needs to handle traits that it knows how to process
+> property values for. If a host wishes to determine if any given entity
+> has a particular trait, the ([forthcoming](https://github.com/OpenAssetIO/OpenAssetIO/issues/31))
+> `entityTraits` method should be used.
+
+### `entityTraits`
+
+For each Entity Reference supplied by the Host, the Manager will return
+a TraitSet containing the IDs of all traits possessed by that entity
+that define its type.
+
+This will include all traits required to correctly type that entity,
+regardless of whether the Manager is capable of provided data for all of
+those traits. I.e the Trait Set the entity was registered with if
+created through the API.
+
+E.g. in the above scenario, the Manager would be expected to return `{
+'file', 'image', 'colorMangaed' }` (i.e. those of the
+`ImageSpecification`) for any images, even though it will only ever
+resolve/register `file` trait data.
+
+This is the differentiator between `entityTraits` and `managementPolicy`
+- this queries the type of a specific entity, the other its data policy
```

Comment:
[fix] This renders as a bullet point, doesn't look like that was the intention.

Reply:
FIxed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/792/commits/e25d8e67d0f71dd4ae20d0e92f4799ca9609f863), thanks. Interestingly it rendered fine in my local engine.